optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/tenant/TenantRestService.java

```diff
@@ -66,5 +67,58 @@
     @POST
     @Path("/config/update")
     TenantConfiguration updateTenantConfiguration(@ApiParam(required = true) TenantConfiguration tenantConfiguration);
+    
+    // TODO: Where should this be?
+    @ApiOperation("Get supported timezones")
+    @GET
+    @Path("/supported/timezones")
+    List<ZoneId> getSupportedTimezones();
+    
+    // Workaround for addTenant since request can have at most one body parameter
+    public static class NewTenant {
```

Comment:
Let's refactor the model to deal with this cleanly.

Obviously, we can't send in Tenant, as that needs to remain extremely lightweight (for the list tenants method that is always called). But what if we send in a RosterState and that embeds a Tenant? The TenantConfiguration is not needed to be send in anyway, as all those fields are defaulted and not in the form?

That being said, we need to discuss about TenantConfiguration - that should probably be renamed to RosterParametrization - and maybe we want to merge it with RosterState. Let's discuss after the meeting scheduling meeting.